pkg/config/config_test.go

```diff
@@ -923,14 +923,6 @@ func TestExperimentalOTLP(t *testing.T) {
 		assert.NoError(t, err)
 		checkConf(t, conf)
 	})
```

Comment:
Maybe `TestOTLP` would be clearer than `TestExperimentalOTLP`

Reply:
Good catch! To be honest.. I think we can probably remove this test altogether, since we already test this on the `pkg/otlp/config_test.go` tests.